code/app/build.gradle.kts

```diff
@@ -0,0 +1,60 @@
+import com.adobe.marketing.mobile.gradle.BuildConstants
+
+plugins {
+    id("com.android.application")
+    id("org.jetbrains.kotlin.android")
+    id("com.diffplug.spotless")
+}
+
+configure<com.diffplug.gradle.spotless.SpotlessExtension> {
+    kotlin {
+        target("src/*/java/**/*.kt")
+        ktlint("0.41.0")
+        licenseHeaderFile("../../config/formatter/adobe.header.txt")
```

Comment:
Replace `licenseHeaderFile` with `licenseHeader` and use this [constant](https://github.com/adobe/aepsdk-commons/blob/dev/android/aepsdk-gradle-plugin/aepsdk-gradle-plugin/src/main/kotlin/com/adobe/marketing/mobile/gradle/BuildConstants.kt#L160). You can also remove the header file from your repository.

Reply:
@praveek Would it be possible to make the `Formatting` variables public as well so we can use common Prettier configurations?